Sources/Commands/ToolWorkspaceDelegate.swift

```diff
@@ -155,7 +159,11 @@ class ToolWorkspaceDelegate: WorkspaceDelegate {
             }
         }
 
-        self.outputHandler("Downloaded \(url) (\(duration.descriptionInSeconds))", false)
+        if fetchDetails.fromCache {
+            self.outputHandler("Fetched \(url) (\(duration.descriptionInSeconds))", false)
```

Comment:
IMO without mentioning the cache, "fetched" on its own isn't clear enough that it actually did fetch from cache

```suggestion
            self.outputHandler("Fetched \(url) from cache (\(duration.descriptionInSeconds))", false)
```